From d16771fa30413c39792e4108b173993796eb58fc Mon Sep 17 00:00:00 2001
From: Uri Mashiach <uri.mashiach@compulab.co.il>
Date: Wed, 13 Feb 2019 08:56:03 +0200
Subject: [PATCH 02/25] ucm-imx8m-mini: Fix KConfig warning

The following compilation warning is generated:
choice value used outside its choice group

Rename the following variables to fix the warning:
* IMX8M_LPDDR4 to UCM_IMX8M_LPDDR4.
* IMX8M_LPDDR4_VAL to UCM_IMX8M_LPDDR4_VAL.

Signed-off-by: Uri Mashiach <uri.mashiach@compulab.co.il>
---
 board/compulab/ucm-imx8m-mini/Kconfig  | 6 +++---
 board/compulab/ucm-imx8m-mini/Makefile | 2 +-
 board/compulab/ucm-imx8m-mini/spl.c    | 8 ++++----
 3 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/board/compulab/ucm-imx8m-mini/Kconfig b/board/compulab/ucm-imx8m-mini/Kconfig
index 02b0e86..1da11b2 100644
--- a/board/compulab/ucm-imx8m-mini/Kconfig
+++ b/board/compulab/ucm-imx8m-mini/Kconfig
@@ -11,16 +11,16 @@ config SYS_CONFIG_NAME
 
 choice
 	prompt "DDR4 Init approach"
-	default IMX8M_LPDDR4_VAL
+	default UCM_IMX8M_LPDDR4_VAL
 	help
 	 DDR4 configuration approach
 
-config IMX8M_LPDDR4
+config UCM_IMX8M_LPDDR4
 	bool "DDR4_INIT_EVK"
 	help
 	 imx8mm_evk board configuration approach
 
-config IMX8M_LPDDR4_VAL
+config UCM_IMX8M_LPDDR4_VAL
 	bool "DDR4_INIT_VAL"
 	help
 	 imx8mm_val board configuration approach
diff --git a/board/compulab/ucm-imx8m-mini/Makefile b/board/compulab/ucm-imx8m-mini/Makefile
index e335f9f..b092c28 100644
--- a/board/compulab/ucm-imx8m-mini/Makefile
+++ b/board/compulab/ucm-imx8m-mini/Makefile
@@ -8,7 +8,7 @@ obj-y += ucm-imx8m-mini.o
 
 ifdef CONFIG_SPL_BUILD
 obj-y += spl.o
-ifdef CONFIG_IMX8M_LPDDR4
+ifdef UCM_CONFIG_IMX8M_LPDDR4
 obj-y += lpddr4_timing.o
 else
 obj-y += ddr/
diff --git a/board/compulab/ucm-imx8m-mini/spl.c b/board/compulab/ucm-imx8m-mini/spl.c
index 6d9cccd..34f7683 100644
--- a/board/compulab/ucm-imx8m-mini/spl.c
+++ b/board/compulab/ucm-imx8m-mini/spl.c
@@ -20,9 +20,9 @@
 #include <fsl_esdhc.h>
 #include <mmc.h>
 
-#ifdef CONFIG_IMX8M_LPDDR4
+#ifdef CONFIG_UCM_IMX8M_LPDDR4
 #include <asm/arch/imx8m_ddr.h>
-#else /* CONFIG_IMX8M_LPDDR4_VAL */
+#else /* CONFIG_UCM_IMX8M_LPDDR4_VAL */
 #include "ddr/ddr.h"
 #endif
 
@@ -30,10 +30,10 @@ DECLARE_GLOBAL_DATA_PTR;
 
 void spl_dram_init(void)
 {
-#ifdef CONFIG_IMX8M_LPDDR4
+#ifdef CONFIG_UCM_IMX8M_LPDDR4
 	/* ddr train */
 	ddr_init(&lpddr4_timing);
-#else /* CONFIG_IMX8M_LPDDR4_VAL */
+#else /* CONFIG_UCM_IMX8M_LPDDR4_VAL */
 	/* ddr train */
 	ddr_init();
 #endif
-- 
1.9.1

